code/_helpers/global_lists.dm

```diff
@@ -113,8 +116,10 @@ var/global/list/string_slot_flags = list(
 	"holster" = SLOT_HOLSTER
 )
 
-var/global/list/current_antags = list()
-var/global/list/current_factions = list()
+//A list of slots where an item doesn't count as "worn" if it's in one of them
+var/global/list/unworn_slots = list(slot_l_hand,slot_r_hand, slot_l_store, slot_r_store, slot_l_store)
```

Comment:
May be robotic slots should be `unworn` too

Reply:
agreed, will do